gtk-3.22/gtk-widgets.css

```diff
@@ -1909,88 +1909,54 @@ menubar:backdrop,
     background-color: transparent;
 }
 
-menubar menuitem,
-.menubar .menuitem {
-    padding: 3px 6px;
-}
-
-menu menuitem,
+menuitem,
 modelbutton,
-popover .menuitem,
-.menu .menuitem,
-.popover .menuitem {
-    padding: 6px;
+.menuitem {
+    padding: 3px 6px;
     text-shadow: none;
-    border-width: 1px 0;
     -gtk-icon-shadow: none;
 }
 
-modelbutton {
-    min-width: 150px;
+menuitem:active,
+menuitem:hover,
+modelbutton:active,
+modelbutton:hover,
+.menuitem:active,
+.menuitem:hover {
+    background-color: alpha (@text_color, 0.15);
 }
 
-popover .menuitem,
-popover .menuitem:disabled {
-    background-image: none;
-    border-color: transparent;
-    box-shadow: none;
+menuitem *:disabled,
+modelbutton *:disabled,
+.menuitem *:disabled {
+    color: @insensitive_color;
+}
+
+menuitem accelerator {
+    color: alpha (@text_color, 0.5);
 }
 
-menu menuitem arrow,
-.menu .menuitem.arrow {
+menuitem arrow {
     -gtk-icon-source: -gtk-icontheme("pan-end-symbolic");
     min-height: 16px;
     min-width: 16px;
 }
 
-menu menuitem arrow:dir(rtl),
-.menu .menuitem.arrow:dir(rtl) {
+menuitem arrow:dir(rtl) {
     -gtk-icon-source: -gtk-icontheme("pan-end-symbolic-rtl");
 }
 
-menu menuitem label,
-menu menuitem image,
+menuitem label,
+menuitem image,
 modelbutton label,
-popover menuitem label,
-popover menuitem image {
-    padding: 0 3px;
-}
-
-popover .menuitem label,
-popover .menuitem image {
+.menuitem label,
+.menuitem image {
     padding: 3px;
 }
 
-menu menuitem:active,
-menu menuitem:hover,
-modelbutton:active,
-modelbutton:hover,
-popup .menuitem:active,
-popup .menuitem:hover,
-popover .menuitem:active,
-popover .menuitem:hover,
-.menu .menuitem:active,
-.menu .menuitem:hover,
-.popup .menuitem:active,
-.popup .menuitem:hover,
-.popover .menuitem:active,
-.popover .menuitem:hover {
-    background-color: alpha (@text_color, 0.15);
-}
-
-popover .menuitem *:disabled,
-.menu .menuitem *:disabled,
-.popover .menuitem *:disabled {
-    color: @insensitive_color;
-}
-
-.menuitem .accelerator {
-    color: alpha (@text_color, 0.4);
-}
-
-.menuitem .accelerator:hover,
-.menuitem .accelerator:active {
-    color: alpha (@fg_color, 0.3);
+modelbutton {
```

Comment:
Is it right to select just the modelbutton here? Considering we're still in the menu part of the stylesheet. The properties set on it do not seem to be consistent with other button selector properties in the rest of the stylesheet.